public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -182,7 +182,12 @@ export const ServiceAccountPageUnconnected = ({
           <Stack justifyContent="space-between" height="auto">
             <h3>Tokens</h3>
             {!serviceAccount.isExternal && (
-              <Button onClick={() => setIsTokenModalOpen(true)} disabled={tokenActionsDisabled}>
+              <Button
+                onClick={() => setIsTokenModalOpen(true)}
+                disabled={tokenActionsDisabled}
+                key="add-service-account-token"
+                icon="plus"
```

Comment:
From what I saw, not all the buttons in Grafana that start with `Add ...` and are located on the right side (as this one is) have the plus icon in front. One example is the `Add service account` button from the `/org/serviceaccounts` page.

Reply:
We also have Create X in grafana. So just wanted to make the page consistent for now